The only places where global_callbacks is actually used is in a module which
declares it "static".  Remove it from this file to avoid that conflict.
--- cyrus-sasl-2.1.20/lib/saslint.h	2005-03-02 02:08:22.200866937 -0500
+++ cyrus-sasl-2.1.20/lib/saslint.h	2005-03-02 02:08:25.793397780 -0500
@@ -109,7 +109,7 @@
   const char *appname;
 } sasl_global_callbacks_t;
 
-extern sasl_global_callbacks_t global_callbacks;
+/* extern sasl_global_callbacks_t global_callbacks; */
 
 typedef struct _sasl_external_properties 
 {
